### PR TITLE
skip fieldsList_e init if it already has fields

### PR DESCRIPTION
**Pull request by mituritsyn**
Fix for small issue when "Disabled fields" list is stacked on each opening

image

### CODE REVIEW

**Review by haslinghuis — DISMISSED** [review]

**Comment by nerdCopter** [comment]
- i dont think this is the proper fix. if you open LOG A with no disabled fields, then open LOG B with some disabled fields, it simply keeps LOG A results.
- proper fix would be clear the results, then continue the normal process.

**Comment by mituritsyn** [comment]
> * i dont think this is the proper fix. if you open LOG A with no disabled fields, then open LOG B with some disabled fields, it simply keeps LOG A results.
> * proper fix would be clear the results, then continue the normal process.

thanks for the tip. with emptying the list there is no issues with different logs in one session

File: js/header_dialog.js [hunk 1]
```diff
@@ -466,6 +466,7 @@ function HeaderDialog(dialog, onSave) {
         ];
 
         const fieldsList_e = $('.fields_list');
+        fieldsList_e.empty();
```
Comment (haslinghuis) [inline js/header_dialog.js:469]:
```suggestion
        const fieldsList_e = $('.fields_list').empty();
```

**Review by nerdCopter — APPROVED** [review]
thank you both. committed @haslinghuis ' suggestion and approved.

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**